pyproject.toml

```
@@ -0,0 +1,28 @@
+[project]
+name = "pyconjp2025-camp-tutorial"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.13"
+dependencies = [
+    "fastapi[all]>=0.115.12",
+    "langchain[deepseek]>=0.3.25",
+]
+
+[dependency-groups]
```

```suggestion
[project.optional-dependencies]
```

素pipも考慮するならこれの方がいいですね(インストールのコマンドも少し変更が必要)